test/testUtils/gameWrapper.ts

```diff
@@ -40,6 +47,46 @@ export class GameWrapper {
       if (cb) cb();
     };
 
+    Pokemon.prototype.damage = function (...args) {
+      const pokemon: Pokemon = this;
+      const ret = GameWrapper.originalDamage.apply(pokemon, args);
+
+      const side = pokemon.isPlayer() ? "Player" : "Enemy";
+      const lowHpMoves = [MoveId.FALSE_SWIPE, MoveId.HARD_PRESS];
+      const currentPhase = globalScene.getCurrentPhase();
+      let moveName = "N/A";
+      let moveId = MoveId.NONE;
+      if (currentPhase instanceof MoveEffectPhase) {
```

Comment:
```suggestion
      if (currentPhase.is<MoveEffectPhase>(PhaseType.MOVE_EFFECT)) {
```
should let you import `MoveEffectPhase` as a type